redocly.yaml

```diff
@@ -13,6 +13,15 @@ l10n:
       name: 日本語
 redirects:
   $ref: redirects.yaml
+responseHeaders: 
+  '**':
+    - name: X-Frame-Options
+      value: DENY
```

Comment:
Actually, based on what I'm seeing on MDN, [`X-Frame-Options` is deprecated](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options) and we should be setting things in [`Content-Security-Policy`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) instead.

And `frame-ancestors` isn't supposed to be a header, but is one of the directives that can be embedded in the `Content-Security-Policy` header. So we should probably change that.

Reply:
Thanks for looking into this one and adding the suggested change.